altos-rust/altos-core/src/sched/mod.rs

```diff
@@ -116,7 +148,15 @@ mod tests {
   use test;
 
   #[test]
-  fn test_start_scheduler() {
+  //Board does not have any tasks scheduled on boot
+  fn no_task() {
+    let _g = test::set_up();
+    assert!(test::current_task().is_none());
+  }
+
+  #[test]
+  ///Test to see that a task gets scheduled
```

Comment:
Should be using the same type of comment here as for other tests (// instead of ///)